shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/main/java/org/apache/shardingsphere/scaling/distsql/handler/CheckScalingQueryResultSet.java

```diff
@@ -36,16 +37,18 @@
  */
 public final class CheckScalingQueryResultSet implements DistSQLResultSet {
     
+    private static final RuleAlteredJobAPI RULE_ALTERED_JOB_API = PipelineJobAPIFactory.getRuleAlteredJobAPI();
```

Comment:
not sure if this is a good naming .

Reply:
Keep the same name as `RuleAlteredJobAPI` interface, it fits Java code conduct.

And `RuleAlteredJobAPI` might be renamed later, it need more consideration and discussion.

Any suggestion on the naming?

Reply:
Wait for an answer.